test/utils.test.ts

```diff
@@ -86,6 +88,12 @@ describe('Utils', () => {
         },
       };
 
+      expect(() => {
+        errorInterceptor(expectedError);
+        fail('should have thrown BasisTheoryApiError');
+      }).toThrow(expectedError);
```

Comment:
Not sure if this asserts every aspect it used to. If it does, please remove commented code.